async/cohttp_async_io.mli

```diff
@@ -0,0 +1,21 @@
+(*
+ * Copyright (c) 2013 Anil Madhavapeddy <anil@recoil.org>
+ *
+ * Permission to use, copy, modify, and distribute this software for
+ * any purpose with or without fee is hereby granted, provided that the
+ * above copyright notice and this permission notice appear in all
+ * copies.  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS
+ * ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE
+ * AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL
+ * DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA
+ * OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
+ * TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ * PERFORMANCE OF THIS SOFTWARE.
+*)
+
+include Cohttp.S.IO
+  with type 'a t = 'a Async.Std.Deferred.t
+   and type ic = Async.Std.Reader.t
+   and type oc = Async.Std.Writer.t
+   and type conn = unit
```

Comment:
@rgrinberg I'm not sure what `conn` is for - I added it because that's what the corresponding lwt module does, and the intention was to bring them into parity. I can remove it if it if it shouldn't be exposed.